src/components/JobDetailCard.jsx

```diff
@@ -0,0 +1,80 @@
+import styled from '@emotion/styled';
+
+// Temporarily importing JSON here to test code
+import data from '../assets/data.json';
+
+const JobDetailContainer = styled.div`
+  padding: ${props => props.theme.layout.space500};
+  margin: 0 auto;
+  border-radius: 6px;
+  background-color: ${props => props.theme.colors.white};
```

Comment:
Can we define a new set of variables under line 49 in `themes/light.js` for all the color definitions? This will make it easier to support a dark theme in the future if we have time. Using `theme` variables will also improve consistency and simplify site-wide color changes.

See an example here: https://github.com/unocorn-squad/job-posting-app?tab=readme-ov-file#creating-styled-components-using-emotion.

Reply:
Great, done! Replaced all the color definitions with component-specific ones in the light theme.